sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/Models/MsTeamsChannelProperties.cs

```diff
@@ -57,35 +48,17 @@ public MsTeamsChannelProperties(bool isEnabled, bool? enableMessaging = default(
         /// </summary>
         partial void CustomInit();
 
-        /// <summary>
-        /// Gets or sets enable messaging for Microsoft Teams channel
-        /// </summary>
-        [JsonProperty(PropertyName = "enableMessaging")]
-        public bool? EnableMessaging { get; set; }
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?